src/types/onyx/Policy.ts

```diff
@@ -235,6 +235,7 @@ type XeroConnectionConfig = OnyxCommon.OnyxValueWithOfflineFeedback<{
     importCustomers: boolean;
     importTaxRates: boolean;
     importTrackingCategories: boolean;
+    syncCustomers: IntegrationEntityMap;
```

Comment:
Please remove `syncCustomers` and use `importCustomers`.
The docs isn't accurate. The correct key is `importCustomers`, and we'll have to update it everywhere

Reply:
I'm getting errors from server, so I can't send the API request, but the server response `syncCustomers`.

Reply:
ohh interesting

Reply:
I see the issue now, let me fix it

Reply:
if I updated it importCustomers API still throw errors so I'm asking for help.

Reply:
If I updated it to `importCustomers` here's the error

<img width="630" alt="Screenshot 2024-05-01 at 20 30 11" src="https://github.com/Expensify/App/assets/16502320/501a35cf-f9db-4a0a-ac59-c4a3feb0f841">

Reply:
@hungvu193 i think it's because 
<img width="828" alt="image" src="https://github.com/Expensify/App/assets/29673073/6ed22f9b-8852-46f5-aa89-f3aca1b4352f">

Reply:
that's BE issue so I don't think it's related, bump @lakchote for helping here

Reply:
Here's the request:
```
curl 'https://dev.new.expensify.com:8082/api/UpdatePolicyConnectionConfiguration?' \
  -H 'Accept: */*' \
  -H 'Accept-Language: en-US,en;q=0.9' \
  -H 'Connection: keep-alive' \
  -H 'Content-Type: multipart/form-data; boundary=----WebKitFormBoundarygHxrwBguyZCnbbdZ' \
  -H 'Cookie: initialReferer=https%3A%2F%2Fdev.new.expensify.com%3A8082%2F; browserGUID=657b1e1ccf98b; mutiny.user.token=88dfd70a-3aa5-4ad9-9080-4f54dbfb73ef; _gcl_au=1.1.1489038878.1709459015; _ga=GA1.1.188411470.1709459015; cf_clearance=M1DirP9FL5i8zejTc7rb_GLuOKdGsyp3cgg9jOJI7rU-1714555619-1.0.1.1-F.I358ic4k57LOYGQU6UmFW1pLakSPDCfFqWltRyb0CZ0yyFgbgxdsmlwThvn4glOQr1Tld4QC0AGAoFQost4Q; _clck=8v84vb%7C2%7Cfle%7C0%7C1523; _uetsid=f5a8bd70079c11efb4b61be823255bec; _uetvid=80f5ea40d94211eea783efc64819d0ae; _clsk=1jc628l%7C1714557292125%7C3%7C1%7Ct.clarity.ms%2Fcollect; _ga_6BR2QJRCCD=GS1.1.1714555619.10.1.1714557292.60.0.0; _cfuvid=onvcaqxpOn013xLmz_Y_h0WpFucfhvDpjK4TEC1k75s-1714568731274-0.0.1.1-604800000; __cf_bm=58TeQrCPXirZSdhJBZkR1rHYuF89iKk9FZrMPLwQzcE-1714572465-1.0.1.1-RcyXwMqSPocs9YpED92uIGK8EObzH5o0fev3LWMVPuxVDR_aR.B4vvdo7_5VMQwZBkZybZ3AOy4.akQlxwlG5Q' \
  -H 'Origin: https://dev.new.expensify.com:8082' \
  -H 'Referer: https://dev.new.expensify.com:8082/settings/workspaces/67D374BFA1B0F027/accounting/xero/import' \
  -H 'Sec-Fetch-Dest: empty' \
  -H 'Sec-Fetch-Mode: cors' \
  -H 'Sec-Fetch-Site: same-origin' \
  -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36' \
  -H 'sec-ch-ua: "Chromium";v="124", "Google Chrome";v="124", "Not-A.Brand";v="99"' \
  -H 'sec-ch-ua-mobile: ?0' \
  -H 'sec-ch-ua-platform: "macOS"' \
  --data-raw $'------WebKitFormBoundarygHxrwBguyZCnbbdZ\r\nContent-Disposition: form-data; name="policyID"\r\n\r\n67D374BFA1B0F027\r\n------WebKitFormBoundarygHxrwBguyZCnbbdZ\r\nContent-Disposition: form-data; name="connectionName"\r\n\r\nxero\r\n------WebKitFormBoundarygHxrwBguyZCnbbdZ\r\nContent-Disposition: form-data; name="settingName"\r\n\r\nimportCustomers\r\n------WebKitFormBoundarygHxrwBguyZCnbbdZ\r\nContent-Disposition: form-data; name="settingValue"\r\n\r\nfalse\r\n------WebKitFormBoundarygHxrwBguyZCnbbdZ\r\nContent-Disposition: form-data; name="idempotencyKey"\r\n\r\nimportCustomers\r\n------WebKitFormBoundarygHxrwBguyZCnbbdZ\r\nContent-Disposition: form-data; name="appversion"\r\n\r\n1.4.69-0\r\n------WebKitFormBoundarygHxrwBguyZCnbbdZ\r\nContent-Disposition: form-data; name="apiRequestType"\r\n\r\nwrite\r\n------WebKitFormBoundarygHxrwBguyZCnbbdZ\r\nContent-Disposition: form-data; name="pusherSocketID"\r\n\r\n708991.54308\r\n------WebKitFormBoundarygHxrwBguyZCnbbdZ\r\nContent-Disposition: form-data; name="shouldRetry"\r\n\r\ntrue\r\n------WebKitFormBoundarygHxrwBguyZCnbbdZ\r\nContent-Disposition: form-data; name="canCancel"\r\n\r\ntrue\r\n------WebKitFormBoundarygHxrwBguyZCnbbdZ\r\nContent-Disposition: form-data; name="authToken"\r\n\r\nAE80628CBA1BD23729B5986BDC63B98B807C33BF8C268A74CA95847D3ABA55BF75DE93D7C9A5ADC8D72676FDC130C1F765717BEBEBFF894EBEE5CD15BA2EBB8C7CDE486549F1E2CFE99361A57973938EEDF74E4F26996768CA32109FB28DE4D51B9DD33FEDCC5A804ECF658DC9B4718CDD1893766D3DBE718971ECC10B6973CFEBC8FDBA80FE1EC9A1A2871DD6FF5DBDAC33DA2A899E17EE74701773482BCC36CEFCA335FFB1A6976D4187FA7CCF87388B642F8DBAEDBD3C4ACA09686CFF24FDE6C8B2B96167B37A179456265E63985865176CE461AC6CBF6FF000C295AF538DF6947F5D2B401A14BCCB6D7790293919197F9DC445260B9CA84658AC0A9CD932D41DA070BA547BA4294E5184E18FFC05E7F386D025AC26D89AB449A0591A2036FA01131D6E9A05804DF210192B7B39AED0EA35B960CCB3B2BEFAD8C2B0FB769CDDF4936D7174D8354E5071EBE77E4029A06ACEC187A5FBFCD155CB6CFE990AB2\r\n------WebKitFormBoundarygHxrwBguyZCnbbdZ\r\nContent-Disposition: form-data; name="referer"\r\n\r\necash\r\n------WebKitFormBoundarygHxrwBguyZCnbbdZ\r\nContent-Disposition: form-data; name="platform"\r\n\r\nweb\r\n------WebKitFormBoundarygHxrwBguyZCnbbdZ\r\nContent-Disposition: form-data; name="api_setCookie"\r\n\r\nfalse\r\n------WebKitFormBoundarygHxrwBguyZCnbbdZ\r\nContent-Disposition: form-data; name="email"\r\n\r\nhungvv193@gmail.com\r\n------WebKitFormBoundarygHxrwBguyZCnbbdZ\r\nContent-Disposition: form-data; name="isFromDevEnv"\r\n\r\ntrue\r\n------WebKitFormBoundarygHxrwBguyZCnbbdZ--\r\n'
```

Reply:
It's a backend issue. The correct key (`importCustomers`) need be added as a valid connection key.

Reply:
Thanks @lakchote , Should we hold this PR for BE changes?